### ISSUE TITLE
Vue3 setup support

### ISSUE THREAD

**Issue by LongJinCen** [TypeStrong/fork-ts-checker-webpack-plugin#650]
## Feature description
In Vue3 SFC, when I use setup, I lose the type checking for TS

```
<template>
 <span>{{ name }}</span>
<template>
<script lang="ts" setup>
import { ref } from 'vue'
const name = ref('test')
// No error
const vertison: string = 1
</script>
```

**Comment by janwidmer** [comment]
any update, when Vue 3 sfc support might come?

**Comment by LongJinCen** [comment]
solve this probleam by add `configFile` options
```
    new ForkTsCheckerWebpackPlugin({
      typescript: {
        extensions: {
          vue: {
            enabled: true,
            compiler: "@vue/compiler-sfc",
          },
        },
        configFile: path.resolve(__dirname, "../tsconfig.json"),
      },
    }),
```

**Comment by piotr-oles** [comment]
Fixed in #667
